setup_wordpress.sh

```diff
@@ -1,3 +1,90 @@
 #!/bin/bash
+set -e
+echo '################## LEMP Stack Installation ####################'
+echo "------------------ Updating APT ------------------"
+sudo apt-get -y update && sudo apt-get -y upgrade
 
-echo 'This script should install and setup Wordpress'
+echo "------------------ Installing Nginx ------------------"
+sudo apt-get -y install nginx
+
+echo "------------------ Configuring Firewall ------------------"
+sudo ufw allow 'Nginx Full'
+
+echo "------------------ Installing MYSQL ------------------"
+sudo apt-get -y install mysql-server
+
+echo "------------------ Creating WordPress Database ------------------"
+mysql_password=QpWo#2LuQ
```

Comment:
I would recommend quoting strings with special characters
```
mysql_password='QpWo#2LuQ'
```

Reply:
I'll add it